engines/router/missionctl/internal/safe_chan.go

```diff
@@ -63,11 +63,6 @@ func (sc *SafeChan) WriteAsync(data interface{}) {
 func (sc *SafeChan) Close() {
 	close(sc.closingCh)
 
-	go func() {
-		for range sc.ch {
-		}
-	}()
-
```

Comment:
Removed this since the linter complained that this go loop isn't doing anything.